src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/JudicialDecisionHandler.java

```diff
@@ -272,8 +272,10 @@ private CallbackResponse checkInputForNextPage(CallbackParams callbackParams) {
         UserInfo userDetails = idamClient.getUserInfo(callbackParams.getParams().get(BEARER_TOKEN).toString());
         caseDataBuilder.judgeTitle(IdamUserUtils.getIdamUserFullName(userDetails));
 
-        if (caseData.getApplicationIsCloaked() == null) {
+        if (caseData.getApplicationIsCloaked() == null && !gaForLipService.isGaForLip(caseData)) {
```

Comment:
this can be refactored 

if (caseData.getApplicationIsCloaked() == null) {
   if (gaForLipService.isGaForLip(caseData)) {
     caseDataBuilder.applicationIsCloaked(helper.isLipApplicationCreatedWithoutNoticeByApplicant(caseData));
   } else {
    caseDataBuilder.applicationIsCloaked(helper.isApplicationCreatedWithoutNoticeByApplicant(caseData));
   }
}